azavea/echo-locator
# UI polish

## Overview

Refine styles of recent feature updates, including:
- Logo
- Translated strings
- Listings (toggles, markers, popups, routing, errors)

### Demo

#### Logo

![image](https://user-images.githubusercontent.com/128699/150000454-8e12f046-7a48-48b8-b1ab-c69dad4f17a7.png)

![image](https://user-images.githubusercontent.com/128699/150000495-0c7f4e90-df99-46c5-b861-0f1a6e1f8700.png)


#### Translations

![image](https://user-images.githubusercontent.com/128699/150001010-a90f58fd-bf7e-4de1-ab52-c3fdfc85ebcc.png)


#### Listing toggles and markers

![image](https://user-images.githubusercontent.com/128699/150001463-0951fb97-f4a1-4381-ae68-c570112423ab.png)


#### Listing popup

![image](https://user-images.githubusercontent.com/128699/150001540-47df5d43-dfa5-40da-9237-52d042a7837d.png)


#### No routes

![image](https://user-images.githubusercontent.com/128699/150001596-8abfa814-ddc9-4986-9003-f777e4acbc4b.png)


### Notes

- I opportunistically changed rent amounts to render with commas.
- In the detail sidebar, I had to split the stats and rent card into two cards to accommodate longer strings from translation. See screenshot above.
- I changed the listing markers to use the brand color from the new logo.
- I replaced the listing toggle buttons with checkboxes, which will require new translations for `ApartmentsToggles` (formerly `Apartments`): "Show apartments from" #398 
  - This also involved deprecating some strings: `HideBHAApartments`, `HideRealtorApartments`, `ShowBHAApartments`, `ShowRealtorApartments`
- I updates some error messages, which will require new translations #398:
  - `ListingsFetchError` "Could not fetch listings"
  - `TripsEmpty`: "No routes found to that destination"



## Testing Instructions

 * Open site when logged out to confirm new logo on auth page
 * Confirm new logo in main header
 * Cycle through all languages on all pages to confirm every component's layout adapts reasonably to string lengths
 * Confirm listing toggle checkboxes work correctly
 * Confirm popup looks right
   *  Including when a listing lacks photos. You can simulate by deleting the `<img>` from a popup or adding `display:none` to its CSS.
 * Confirm "No routes found…" message in sidebar when attempting to route to a distant destination (eg, a listing in _Haverhill – 01832_ when using Transit).


Resolves #388
